src/Environment.php

```diff
@@ -55,7 +55,7 @@ public function getPayoutUrl(): string
     private function getBaseUrl(): string
     {
         return match ($this) {
-            self::LIVE => 'https://backend.flexpay.cd/api/rest/v1',
+            self::LIVE => 'http://ip:port/api/rest/v1/merchantPayOutService',
```

Comment:
https://backend.flexpay.cd/api/rest/v1

Reply:
url de payout pour les deux environnement !

sprintf('%s/merchantPayOutService', $this->getBaseUrl())